<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureScoutToolBar</class>
 <widget class="QToolBar" name="FeatureScoutToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>42</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>IFV_ToolBar</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <action name="actionMultiRendering">
   <property name="text">
    <string>Multi Rendering</string>
   </property>
   <property name="toolTip">
    <string>Multi Class Rendering</string>
   </property>
  </action>
  <action name="actionLength_Distribution">
   <property name="text">
    <string>Length Distribution</string>
   </property>
   <property name="toolTip">
    <string>Length Distribution Rendering</string>
   </property>
  </action>
  <action name="actionMeanObject">
   <property name="text">
    <string>Mean Object</string>
   </property>
   <property name="toolTip">
    <string>Mean Object Rendering</string>
   </property>
  </action>
  <action name="actionOrientation_Rendering">
   <property name="text">
    <string>Orientation Rendering</string>
   </property>
   <property name="toolTip">
    <string>Orientation Distribution Rendering</string>
   </property>
  </action>
  <action name="actionActivate_SPM">
   <property name="text">
    <string>Activate SPM</string>
   </property>
   <property name="toolTip">
    <string>Activate Scatterplot Matrix</string>
   </property>
  </action>
  <action name="actionSettingsPC">
   <property name="text">
    <string>PC settings</string>
   </property>
   <property name="toolTip">
    <string>Open Parallel Coordinates Settings Dialog</string>
   </property>
  </action>
  <addaction name="actionLength_Distribution"/>
  <addaction name="actionOrientation_Rendering"/>
  <addaction name="actionMultiRendering"/>
  <addaction name="actionMeanObject"/>
  <addaction name="actionActivate_SPM"/>
  <addaction name="actionSettingsPC"/>
 </widget>
 <resources/>
 <connections/>
</ui>
